lib/insert-link.js

```diff
@@ -19,6 +20,12 @@ function createLinkElement(text, dataAttr = {}) {
     }
   }
 
+  if (dataAttr.resolver == 'liveResolverQuery') {
+    const type = dataAttr.type;
+    const target = dataAttr.target;
+    linkEl.href = liveResolverQuery({type, target}, true).url;
+  }
```

Comment:
In same cases the liveResolver have to be called multiple times for the same target. For example in a JavaScript file like `require('foo')` the target `foo` could be either a bower or npm module.

Maybe the [live-resolver](https://github.com/OctoLinker/live-resolver) should accept multiple types like `https://githublinker.herokuapp.com/q/?type=npm,bower&target=foo`